ansible/roles/test/tasks/everflow_testbed/everflow_main.yml

```diff
@@ -0,0 +1,37 @@
+- fail:
+    msg: "Mirror stage is not defined or invalid"
+  when: mirror_stage is not defined or mirror_stage not in ['ingress', 'egress']
+
+- fail:
+    msg: "ACL stage is not defined or invalid"
+  when: acl_stage is not defined or acl_stage not in ['ingress', 'egress']
+
+- name: Set flag that recover from config_db.json is needed (default - false)
+  set_fact:
+    recover_from_cfgdb_file: False
+
+- name: Apply Everflow configuration.
+  include: "roles/test/tasks/everflow_testbed/apply_config.yml"
+  tags: everflow_tb_configure
+
+- name: Run Everflow tests.
+  include: "roles/test/tasks/everflow_testbed/run_test.yml"
+  vars:
+    dst_port_type: "tor"
+  tags: everflow_tb_test
+
+- name: Run Everflow tests.
+  include: "roles/test/tasks/everflow_testbed/run_test.yml"
+  vars:
+    dst_port_type: "spine"
+  tags: everflow_tb_test
+
+- name: Clear Everflow configuration.
+  include: "roles/test/tasks/everflow_testbed/del_config.yml"
+  tags: everflow_tb_cleanup
+
+- name: Reload config
```

Comment:
not sure i fully understand. previously it was mentioned that a variable is kept if this need to be done or not but i dont see this variable is being used.

Reply:
This task has a ```when: recover_from_cfgdb_file``` condition